DocGen4/Output.lean

```diff
@@ -90,15 +105,20 @@ def htmlOutputResults (baseConfig : SiteBaseContext) (result : AnalyzerResult) (
       depthToRoot := modName.components.dropLast.length
       currentName := some modName
     }
-    let moduleHtml := moduleToHtml module |>.run config baseConfig
+    let (moduleHtml, backrefs) := moduleToHtml module #[] |>.run config baseConfig
     FS.createDirAll fileDir
     FS.writeFile filePath moduleHtml.toString
+    FS.writeFile (declarationsBasePath / s!"backrefs-{module.name}.json") (toString (toJson backrefs))
```

Comment:
Is the motivation for storing backrefs in a separate JSON file to not blow up the size of `declaration-data.bmp` even more or is there something else?

Reply:
You are misunderstanding me, I think that it's great that it does not need to be part of declaration-data.bmp, I was merely wondering if there was a reason besides the size concern that you did this decision.